ethcore/src/block.rs

```diff
@@ -137,6 +142,7 @@ impl ExecutedBlock {
 			transactions_set: Default::default(),
 			state: state,
 			traces: if tracing {Some(Vec::new())} else {None},
+			tracing_enabled: tracing,
```

Comment:
Yes, we can. I thought about it. But IMHO relying on the explicit flag is better than basing on the assumption. The logic of working with traces can be changed and this assumption may become false